website/blog/2023-06-11-databend-weekly-97.md

```diff
@@ -52,15 +52,23 @@ select $1, $2, a, b from t1;
 └─────────────────────────────────┘
 ```
 
-We have also added column position support for `SELECT FROM` with NDJSON, and support for other formats is in progress.
+You can also use column positions when you SELECT FROM a staged NDJSON files. We are also actively working on extending this support to other formats. When using the COPY INTO statement to copy data from a stage, the matching is performed based on the field names rather than column positions.
 
 ```sql
 select $1 from @my_stage
 
 copy into my_table from (select $1 from @my_stage t)
 ```
 
-Please note that when using `select` for NDJson, only `$1` is allowed. This stands for the entire line and has a variant type. On the other hand, when copying into `my_table` from `@my_stage`, matching is done by name of top-level fields.
+It is important to note that when using the SELECT statement for NDJSON in Databend, only $1 is allowed, representing the entire row and having the data type variant. 
+
+```sql
+-- Select the entire row using column position:
+SELECT $1 FROM @my_stage
+
+--Select a specific field named "a" using column position:
+SELECT $1:a FROM @my_stage
```

Comment:
need to specify file_format.
either when `create stage` or when select/copy, note copy with transform and select has different way to do it:

```
SELECT $1:a FROM @my_stage (file_format=>'ndjson')
copy into my_table from (select $1 from @my_stage t) FILE_FORMAT = (type = NDJSON)
```

Reply:
rest lgtm